src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -119,5 +185,353 @@ class QBridgeMIBObjects(metaclass=MIBMeta, prefix='.1.3.6.1.2.1.17.7.1'):
 
     fdb_updater = FdbUpdater()
 
+    dot1qVlanVersionNumber = \
+        MIBEntry('1.1.0', ValueType.INTEGER, fdb_updater.get_vlan_version)
+
+    dot1qMaxVlanId = \
+        MIBEntry('1.2.0', ValueType.INTEGER, fdb_updater.get_MaxVlanId)
+
+    dot1qMaxSupportedVlans = \
+        MIBEntry('1.3.0', ValueType.GAUGE_32, fdb_updater.get_MaxSupportedVlans)
+
+    dot1qNumVlans = \
+        MIBEntry('1.4.0', ValueType.GAUGE_32, fdb_updater.get_dot1qNumVlans)
+
     dot1qTpFdbPort = \
         SubtreeMIBEntry('2.2.1.2', fdb_updater, ValueType.INTEGER, fdb_updater.fdb_ifindex)
+
+    dot1qTpFdbStatus = \
+        SubtreeMIBEntry('2.2.1.3', fdb_updater, ValueType.INTEGER, fdb_updater.fdb_status)
+
+class FdbTableUpdater(MIBUpdater):
+    def __init__(self):
+        super().__init__()
+        self.db_conn = mibs.init_db()
```

Comment:
Currently we do not have support for the exising FDB MIB for multi-asic platform.
But will be good to keep it in sync to use Namespace.init_namespace_dbs() instead of mibs.init_db().
Same is used in fdb_updater as well.
https://github.com/Azure/sonic-snmpagent/blob/master/src/sonic_ax_impl/mibs/ietf/rfc4363.py#L12

Reply:
we will take care